src/plugins/spdk/spdk_env/memory.c

```
@@ -0,0 +1,80 @@
+#include "spdk/memory.h"
+
+#include "env_internal.h"
+#include "env_mem_map.h"
+
+
+
+#ifdef __linux__
+#include <numaif.h>
+#endif
+
+uint64_t spdk_vtophys(const void *buf, uint64_t *size)
```

Cppcheck (2.21.0)

[style] The function 'spdk_vtophys' is never used.

(unusedFunction)